lib/ansible/modules/windows/win_mssql_login.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Daniele Lazzari <lazzari@mailup.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# This is a windows documentation stub.  Actual code lives in the .ps1
+# file of the same name.
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: win_mssql_login
+version_added: '2.7'
+short_description: adds or removes a MSSQL Server login
+description:
+    - Adds or removes MSSQL Server login
+requirements:
+  - Powershell module sqlps or Powershell module SqlServer on the remote host
+options:
```

Comment:
Each option entry should be a full sentence, i.e. it starts with a capital and ends with a full stop.